string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,10 @@
+/* exported isVowel */
+/* -Check to see if the input is a vowel
+  -if not, return false */
+
+function isVowel(char) {
+  if ((char.toUpperCase(char) === 'A') || (char.toUpperCase(char) === 'E') || (char.toUpperCase(char) === 'I') || (char.toUpperCase(char) === 'O') ||
```

Comment:
You should call `toUpperCase` once and save the value before the conditional, so you don't need to do that action over and over in each expression of the condition.